utility/fileExistence.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const path = require('path');
+
+exports.fileExistence = ({ file, params }) => {
+  const { username, repo } = params;
```

Comment:
this functions appends metadata of file upload and modification by the user in respective repository
metadata includes
    1. reponame
    2. filename
    3. date and time of modification